back/node_watcher/src/tasks/index.ts

```diff
@@ -6,25 +6,36 @@ import createBlockNumber from './createBlockNumber';
 import createEra from './createEra';
 import createMotion from './createMotion';
 import createMotionStatus from './createMotionStatus';
+import createNominationAndValidators from './createNominationAndValidators';
+import createOfflineValidator from './createOfflineValidator';
 import createPreimage from './createPreimage';
 import createProposal from './createProposal';
 import createProposalStatus from './createProposalStatus';
 import createReferendum from './createReferendum';
 import createReferendumStatus from './createReferendumStatus';
+import createReward from './createReward';
 import createSession from './createSession';
 import createSlashing from './createSlashing';
+import createStake from './createStake';
 import createTotalIssuance from './createTotalIssuance';
-import createValidator from './createValidator';
 import { NomidotTask } from './types';
 
 // N.B. Order of tasks matters here
 export const nomidotTasks: NomidotTask[] = [
   createBlockNumber,
   createSession,
+  createStake,
+  createOfflineValidator,
+  createReward,
   createEra,
   createSlashing,
   createTotalIssuance,
-  createValidator,
+  createNominationAndValidators,
+  createPreimage,
+  createProposal,
+  createProposalStatus,
+  createReferendum,
+  createReferendumStatus,
```

Comment:
These are duplicated afaics. Weird that the CI didn't catch that.

Reply:
🙏🙏🙏 holy cow that would have sucked if you hadn't caught that